app/account/page.tsx

```diff
@@ -0,0 +1,143 @@
+'use client'
+
+import {
+  Heading,
+  Flex,
+  Box,
+  Container,
+  VStack,
+  Image,
+  Text,
+  useColorModeValue,
+  Spacer
+} from '@chakra-ui/react'
+import { CiEdit } from 'react-icons/ci'
+import { FaRegTrashAlt } from 'react-icons/fa'
+import { MdAccountCircle } from 'react-icons/md'
+import { PrimaryButton, AlertButton } from '@/components/button'
+import { Link } from '@/components/link'
+import { Loading } from '@/components/loading'
+import { useGetUserQuery } from '@generated/api'
+
+export default function AccountPage() {
+  const bg = useColorModeValue('white', 'gray.800')
+  const color = useColorModeValue('black', 'gray.300')
+  // TODO: Remove this line after implementing login
+  const tmpUuid = 'a1f4509d-39f6-4de7-921d-201ec3d4578f'
```

Comment:
Because this PR is merged, you can actually change the UUID from context!
https://github.com/tabi-memo/frontend/pull/18

Reply:
I changed from temporary uuid to real uuid :pray:
https://github.com/tabi-memo/frontend/pull/13/commits/baaa7f575d92dde2b52bb383fcd8675417f6d46b